tests-ng/plugins/utils.py

```diff
@@ -1,14 +1,81 @@
+from pathlib import Path
+from typing import Dict, List, Optional
+
 import pytest
 
 # Various utility functions to make tests more readable
 # This should not contain test-assertions, but only abstract details that make tests harder to read
 
-def equals_ignore_case(a: str, b: str) -> str:
+
+def equals_ignore_case(a: str, b: str) -> bool:
```

Comment:
good catch, how did this not show up before? would have assumed the editor shows a warning in this case